src/app/backend/call.rs

```diff
@@ -0,0 +1,151 @@
+use super::State;
+use crate::{
+    app::{
+        backend::state::{check_for_bot_response, Inner},
+        Event, TurnToSpeak,
+    },
+    aws_api::{speech_to_text, text_to_speech, upload_speech},
+    message::create_prompt_from_messages,
+    openai_api::fetch_response_to_prompt,
+};
+use std::{mem, time::Instant};
+use tokio::sync::oneshot;
+use tracing::{debug, trace};
+
+pub fn handle_backend_events(state: &mut State) -> Result<(), anyhow::Error> {
```

Comment:
What is this?

Reply:
Oh weird! Because I'm on a branch that's in PR right now, I can comment in VS Code and it must be publishing these comments to GitHub! That's so cool!